src/components/Confirm/style.css

```diff
@@ -7,12 +9,16 @@
 
 .image {
   max-width: 92%;
+  @media (--small-viewport) {
```

Comment:
You already have one `@media (--small-viewport) {...}` below you can put `max-width` there